iroha/src/torii.rs

```diff
@@ -142,6 +142,79 @@ struct ToriiState {
     sumeragi: Arc<RwLock<Sumeragi>>,
 }
 
+#[derive(Clone, Eq, PartialEq, Debug, Default)]
+struct Pagination {
```

Comment:
I would move it into the iroha_data_model crate, as it can be needed for `iroha_client`